src/plugins/FileService/utils/file.ts

```diff
@@ -0,0 +1,15 @@
+export function formatFileSize(input = 0, si = isApplePlatform()) {
```

Comment:
When input = `2333333333333333433`, it returns `2.0 undefinediB`

When input = -1, it returns `NaN undefinediB`

When input = 1, it returns `1 iB`

Reply:
The not a problem, input range is known.